modules/ROOT/pages/runtime-manager-agent.adoc

```diff
@@ -7,70 +7,74 @@ endif::[]
 image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
 image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
 image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
-The Runtime Manager agent is a plugin extension for Mule that exposes the Mule API. Runtime Manager (both the cloud console and the Anypoint Platform Private Cloud Edition) uses this agent to communicate with its registered Mule instances.
+Anypoint Runtime Manager agent is a Mule runtime engine plug-in that exposes the Mule API. Runtime Manager (both the cloud console managed by MuleSoft and Anypoint Platform Private Cloud Edition) uses this agent to communicate with its registered Mule instances.
 
-Outside the Runtime Manager, you can use the Runtime Manager agent to monitor and control Mule instances by calling Runtime Manager agent APIs from external systems, and/or have Mule publish its own data to external systems. The Runtime Manager agent has many features, including:
+For information about how the Runtime Manager agent communicates with Runtime Manager, see xref:deployment-strategies.adoc[Deployment Strategies].
 
-* Controlling applications, domains and services:
-** List, deploy, undeploy or redeploy domains.
-** List, deploy, undeploy, get status of, start or stop applications.
-* Publishing Mule metrics to external analytics systems.
+Runtime Manager is available only for enterprise versions of Mule, not for community versions. The agent supports Mule 3.6.0 or later.
 
-[NOTE]
-====
-* The Runtime Manager is available only for Enterprise versions of Mule, not for Community versions. The Runtime Manager agent supports Mule 3.6.0 or later.
-* If you're deploying your applications to xref:deploying-to-pcf.adoc[Pivotal Cloud Foundry], then you don't need to worry about installing or configuring the Runtime Manager agent. In this scenario, the Runtime Manager agent functions as usual as the link between Mule instances and the Runtime Manager. However, the same PCF buildpack that automatically creates new runtime instances also creates Runtime Manager agent instances and registers these to the Runtime Manager. If you want to change the configuration of your Runtime Manager agent instances, you must modify the buildpack.
-====
+You can use the agent to monitor and control Mule instances by calling Runtime Manager agent APIs from external systems and having Mule publish its data to external systems. Runtime Manager agent features include:
 
-== Communication Protocols
+* Controlling applications, domains, and services:
+** List, deploy, undeploy, or redeploy domains
```

Comment:
Managing domains is true only when using the agent in REST transport. When registering a runtime to the platform it's not possible to manage domains via Runtime Manager UI